appconfig/settings/tailer.q

```diff
@@ -3,8 +3,9 @@
 \d .wdb
 savedir:hsym `$getenv[`KDBWDB]		// location to save wdb data
 hdbdir:hsym`$getenv[`KDBHDB]            // move wdb database to different location
-sortworkertypes:()			// tailer doesn't need to connect to sortworkers
-
+sortworkertypes:()			// WDB doesn't need to connect to sortworkers
+rowthresh:10000				// table savedown row size threshold
+period:0D00:10:00			// how often to check table sizes against threshold
```

Comment:
these appear to be new config variables - if so, I don't think they are defined in the standard torq way at the top of the process .q (i.e. with `@[value;`rowthresh;...` etc.)

Additionally, the wdb already has variables for these purposes - `numrows` and `settimer`

It seems confusing to me to introduce new variables that do the same thing as those ones, why not just use the existing ones?